phoenicis-tools/src/main/java/org/phoenicis/tools/system/opener/Opener.java

```diff
@@ -0,0 +1,23 @@
+package org.phoenicis.tools.system.opener;
+
+import java.net.URI;
+
+/**
+ * System Opener
+ */
+@FunctionalInterface
+public interface Opener {
+    /**
+     * Open a file
+     * @param file to be opened
+     */
+    void open(String file);
```

Comment:
Understand the idea. But no one can prevent me from calling this with a String URL directly (which basically leads to the same code injection issues like my implementation).